code/game/gamemodes/cult/runes.dm

```diff
@@ -1033,13 +1033,16 @@ var/list/teleport_runes = list()
 	N.mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	new_human.key = ghost_to_spawn.key
 	SSticker.mode.add_cultist(new_human.mind, 0)
+	new_human.mind.special_role = "Cultist"
 	summoned_guys |= new_human
 	ghosts++
 	to_chat(new_human, "<span class='cultitalic'><b>You are a servant of [SSticker.cultdat.entity_title3]. You have been made semi-corporeal by the cult of [SSticker.cultdat.entity_name], and you are to serve them at all costs.</b></span>")
 
 	while(user in get_turf(src))
 		if(user.stat)
 			break
+		if(new_human.z != user.z)
```

Comment:
Use `get_turf(people).z` instead. That way people who are in a locker or something else will not trigger the despawn (because it sets their Z to 0).

Reply:
You might have to typecast it too, not 100% sure. Try without first I guess